### ISSUE TITLE
Keyboard navigation using arrow keys doesn't work in several places

### ISSUE THREAD

**Issue by alexjlockwood** [romannurik/AndroidIconAnimator#127]
For example, if you select a path and try to navigate through the tree of paths using the up/down arrow keys, nothing happens.

On an unrelated note, I would guess that when one of the layer items has focus, there would be a keyboard shortcut to potentially expand/collapse that layer if possible.

**Comment by alexjlockwood** [comment]
(Also if you have any ideas on how to get started implementing this LMK... because I want to add it to ShapeShifter and don't know where to start. I'm an a11y noob. :P Maybe I can submit a PR for you too once I figure it out.)
